app/customer.py

```diff
@@ -0,0 +1,50 @@
+from app.car import Car
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(self, name: str, product_cart: dict,
+                 location: list, money: float, car: Car) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
+        self.money = money
+        self.car = car
+        self.target_shop = None
+
+    def calculate_all_cost(self, shops: list[Shop], fuel_price: float) -> None:
```

Comment:
The method 'calculate_all_cost' has a descriptive name, but consider adding a docstring to explain its purpose and parameters for better readability.